data/iterations.json

```diff
@@ -0,0 +1,134 @@
+[
+    {
+        "from": "2018-12-03",
```

Comment:
Why from / to instead of just a list of two dates?

Reply:
It seemed more semantic to me, but it's arbitrary. I'm happy to change it if you'd like, just confirm and I'll add it to the to-do.